lib/active_stash/assess/column_name_rules.rb

```diff
@@ -0,0 +1,49 @@
+module ActiveStash
+  class Assess
+    # Rules for checking column names for sensitive data.
+    #
+    # @private
+    class ColumnNameRules
+      ERR_PII = "AS0001"
+      ERR_PHI = "AS0002"
+      ERR_FINANCIAL = "AS0003"
+      ERR_AUTH = "AS0004"
+
+      RULES = [
+        { name: 'last_name', display_name: 'names', column_names: %w[name fname firstname lastname lname surname], error_code: ERR_PII },
```

Comment:
I realise the list of rules could end up quite long over time but worth including fields with underscores like `first_name`?

Reply:
Oh I just realised that you normalise the check by removing underscores later. Carry on!